src/integrator.jl

```diff
@@ -162,6 +172,8 @@ end
 
 nom_step_size(lf::JitteredLeapfrog) = lf.ϵ0
 
+update_nom_step_size(lf::JitteredLeapfrog, ϵ0) = reconstruct(lf, ϵ0=ϵ0, ϵ=ϵ0)
```

Comment:
Updating `ϵ` seems like the right thing to do, but it seems a little suspicious to me that if we don't update it, then it appears to never be updated by `jitter`,  so adaptation drives `ϵ0` to 0.

Reply:
I feel the right thing is to update `ϵ0` as you do here. `ϵ` can probably be anything because it will be override in `jitter`.

Reply:
> `ϵ` can probably be anything because it will be override in `jitter`.

That's what I expected, but not updating `ϵ` results in `ϵ0` being adapted to 0, which indicates that `ϵ` is being used for adaptation somewhere before `jitter` is called.

Reply:
I'll see if I can run down why this is happening.

Reply:
Okay, figured it out. `transition` for `StaticTrajectory` was creating a jittered integrator but only used it for reading stats, not for sampling (yikes).

Reply:
@sethaxen Wow. I actually worked with that function in my personal fork and never realized it. Thank god you picked that up.

Reply:
I see. This is a tricky bug. Can we have it fixed in this PR as well?

Also FYI, in my PR for unifying interfaces for different trajectories (#214), I unify the jitter (and also momentum refreshment) [here](https://github.com/TuringLang/AdvancedHMC.jl/blob/kx/unify-trajectory-1/src/trajectory.jl#L220) so that this bug is kind of being dodged. Still waiting for Hong to take a final pass of the PR to get it merged though.

Reply:
> I see. This is a tricky bug. Can we have it fixed in this PR as well?

Yes, I fixed it in 4d09a4b

Ah okay, after this is merged I will review that as well to be sure I understand what it does.

Reply:
Thanks! The fix looks correct to me.

> Ah okay, after this is merged I will review that as well to be sure I understand what it does.

Great. That will be helpful!